support/k8s/k8s-workload-registrar/mode-crd/controllers/endpoint_controller_test.go

```diff
@@ -125,6 +125,6 @@ func (s *EndpointControllerTestSuite) TestAddDNSName() {
 	})
 	s.Require().NoError(err)
 	s.Require().Len(spiffeIDList.Items, 1)
-	s.Require().Len(spiffeIDList.Items[0].Spec.DnsNames, 2)
-	s.Require().Equal("test-endpoint.default.svc", spiffeIDList.Items[0].Spec.DnsNames[1])
```

Comment:
What causes these lines to change?

Reply:
Previously the pod name was already included in the dnsNames list when this test ran. Now it gets added at a different point, and isn't available for this test.